src/components/NotificationsPanel/NotificationsPanel.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { styled, theme, font, BadgeNumber } from '@aragon/ui'
+import NotificationItem from './NotificationItem'
+
+const NotificationsPanel = ({ notifications = [] }) => (
+  <Main>
+    <Header>
+      <Title>
+        <span>Notifications</span>
+        <BadgeNumber number={4} />
+      </Title>
+      <ActionLink>Clear all</ActionLink>
```

Comment:
Potential simplification for this header:

```diff
    <Header>
+     <Title>Notifications</Title>
+     <BadgeNumber number={4} />
      <ActionLink>Clear all</ActionLink>
    </Header>

...

const Header = styled.header`
  flex-shrink: 0;
  display: flex;
- justify-content: space-between;
+ justify-content: flex-start;
  align-items: center;
  width: 100%;
  height: 64px;
  padding: 0 20px;
  border-bottom: 1px solid #e8e8e8;
`

const Title = styled.h1`
- display: flex;
- align-items: center;
  color: ${theme.textSecondary};
+ margin-right: 15px;
  text-transform: lowercase;
  font-variant: small-caps;
  font-weight: 600;
- & > span:first-child {
-   margin-right: 15px;
- }
`

const ActionLink = styled.a`
+ margin-left: auto;
  text-decoration: underline;
  color: ${theme.textTertiary};
  cursor: pointer;
  ${font({ size: 'small' })};
`
```

Reply:
The problem is that it would change the structure from this:

```jsx
<h1>Notifications − 4</h1>
<a>Clear all</a>
```

To this:

```jsx
<h1>Notifications</h1>
<span>4</span>
<a>Clear all</a>
```

And I think “4” should stay next to “notifications”, in the `<h1>` itself, because it is directly related. It should also have a proper title to describe it, like this: 
<img width="255" alt="image" src="https://user-images.githubusercontent.com/36158/33783887-7f3efbb6-dc56-11e7-944c-f60e9efc052e.png">

What do you think?

Reply:
Ah, hmm, I hadn't considered the badge being part of the header.

Yeah, I'd say `<h1>Notifications − 4</h1>` is better.